src/test/resource/fake-user-data-2

```diff
@@ -0,0 +1 @@
+fake user data 2
```

Comment:
I think we should include the contents of this files inside the testing code itself instead of adding the files to the resource folder